doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,103 @@
+.. _User_Guide:
+
+#####################
+`ts_imsim` User Guide
```

Comment:
I think you only need "User Guide". Otherwise, it looks like this:

<img width="624" alt="Screenshot 2023-11-07 at 9 11 44 PM" src="https://github.com/lsst-ts/ts_imsim/assets/11474902/3c88767b-5855-4140-bc2c-d6695f8b3c9f">